### PR TITLE
traitor toxins kit

### PR DESCRIPTION
**Pull request by Iamgoofball**
btw chef special the reagent doesn't exist anymore

it was replaced with amantin, the proper version of it

### CODE REVIEW

**Comment by Krausus** [comment]
> amantin

Looks like the reagent is called `amanitin`, but the code in this commit tries to use `amantin`.

File: code/modules/reagents/reagent_containers/glass/bottle.dm [hunk 1]
```diff
@@ -201,4 +201,58 @@
 	icon_state = "bottle17"
 	New()
 		..()
-		reagents.add_reagent("morphine", 30)
\ No newline at end of file
+		reagents.add_reagent("morphine", 30)
+
+/obj/item/weapon/reagent_containers/glass/bottle/traitor
+	name = "fuck shit Bottle"
+	desc = "A small bottle. Contains woody's got wood in liquid form."
```
Comment (Fox-McCloud) [inline code/modules/reagents/reagent_containers/glass/bottle.dm]: hahah, no =p

File: code/modules/reagents/reagent_containers/glass/bottle.dm [hunk 2]
```diff
@@ -201,4 +201,58 @@
...
+/obj/item/weapon/reagent_containers/glass/bottle/traitor/sodium_thiopental
+	poison = "sodium_thiopental"
+	poison_name = "Sodium Thiopental"
+/obj/item/weapon/reagent_containers/glass/bottle/traitor/amantin
+	poison = "amantin"
+	poison_name = "Amantin"
+/obj/item/weapon/reagent_containers/glass/bottle/traitor/curare
+	poison = "curare"
+	poison_name = "Curare"
+/obj/item/weapon/reagent_containers/glass/bottle/traitor/coiine
```
Comment (Fox-McCloud) [inline code/modules/reagents/reagent_containers/glass/bottle.dm]: name is "coniine" not coiine.

**Comment by Fox-McCloud** [comment]
If you correct the joke descriptions and the spellings of coniine, this can be merged.

**Comment by Fox-McCloud** [comment]
tator uplink

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 3]
```diff
@@ -185,4 +185,23 @@
...
+
+	New()
+		..()
+		new /obj/item/weapon/reagent_containers/glass/bottle/traitor/initropidril(src)
+		new /obj/item/weapon/reagent_containers/glass/bottle/traitor/polonium(src)
+		new /obj/item/weapon/reagent_containers/glass/bottle/traitor/pancuronium(src)
+		new /obj/item/weapon/reagent_containers/glass/bottle/traitor/sodium_thiopental(src)
+		new /obj/item/weapon/reagent_containers/glass/bottle/traitor/amantin(src)
+		new /obj/item/weapon/reagent_containers/glass/bottle/traitor/curare(src)
+		new /obj/item/weapon/reagent_containers/glass/bottle/traitor/coiine(src)
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: just need to fix this and it can be merged.

**Comment by Krausus** [comment]
Amanitin is still misspelled in several places, though.

**Comment by Fox-McCloud** [comment]
^ that too

**Pull request closed without merging**